src/app/donation-complete/donation-complete.component.ts

```diff
@@ -158,6 +159,10 @@ export class DonationCompleteComponent implements OnInit {
           }
         },
         (error: HttpErrorResponse) => {
+          if (error.error?.error?.type === "DUPLICATE_EMAIL_ADDRESS_WITH_PASSWORD") {
```

Comment:
Not critical, but would be nice to have a more type safe way to do this. At the moment it's relying on `any` a lot. If it doesn't work we shouldn't get a crash, it will just show the old error message like it did before this PR.

Reply:
I agree, I don't love these!

Reply:
hmm no of course we would get a crash if one of the objects was not null or undefined but didn't have the property we need. But I can cope, it's unlikely and it's only crashing in what we already know is an error handling task.